dev/workflow/applications/gcafs_forecast_only.py

```diff
@@ -0,0 +1,146 @@
+
+        Parameters
+        ----------
+        conf : Configuration
+            Configuration object containing GCAFS settings
+        """
+        super().__init__(conf)
+
+        base = conf.parse_config('config.base')
+        self.run = base.get('RUN', 'gcafs')
+        self.runs = [self.run]
+
+    def _get_run_options(self, conf: Configuration) -> Dict[str, Any]:
+        """
+        Get run-specific options for GCAFS.
+
+        Parameters
+        ----------
+        conf : Configuration
+            Configuration object containing run settings
+
+        Returns
+        -------
+        Dict[str, Any]
+            Dictionary containing run options including ensemble settings
+        """
+        run_options = super()._get_run_options(conf)
+        run_options[self.run]['nens'] = conf.parse_config('config.base').get('NMEM_ENS', 0)
+        return run_options
+
+    def _get_app_configs(self, run):
+        """
+        Get configuration files involved in GCAFS execution.
+
+        Parameters
+        ----------
+        run : str
+            Name of the run configuration to process
+
+        Returns
+        -------
+        list
+            List of configuration file names needed for the GCAFS run
+        """
+        options = self.run_options[run]
+        configs = ['stage_ic', 'fcst', 'atmos_products', 'aero', 'aerosol_init', 'prep_emissions']
+
+        if options['do_wave']:
+            configs += ['waveinit', 'wavepostsbs', 'wavepostpnt']
+
+        if options['do_archcom']:
+            configs += ['arch_tars']
+            if options['do_globusarch']:
+                configs += ['globus']
+
+        configs += ['arch_vrfy', 'cleanup']
+        return configs
+
+    @staticmethod
+    def _update_base(base_in):
+        """
+        Update base configuration with GCAFS-specific settings.
+
+        Parameters
+        ----------
+        base_in : dict
+            Input base configuration dictionary
+
+        Returns
+        -------
+        dict
+            Updated base configuration with GCAFS settings
+        """
+        base_out = base_in.copy()
+        base_out['RUN'] = 'gcafs'
+        return base_out
+
+    def get_task_names(self):
+        """
+        Get ordered list of tasks for GCAFS workflow.
+
+        This method determines which tasks should be run based on the configuration
+        options, including aerosol forecasting, ensemble processing, and archival tasks.
+
+        Returns
+        -------
+        dict
+            Dictionary with run name as key and list of task names as value
+        """
+        options = self.run_options[self.run]
+        tasks = ['stage_ic']
+
+        if options['do_aero_fcst']:
+            tasks += ['prep_emissions']
+            tasks += ['aerosol_init']
+
+        tasks += ['fcst', 'atmos_prod']
+
+        # Only add ensemble tasks if nens > 0
+        if int(options.get('nens', 0)) > 0:
+            tasks += ['atmos_ensstat']
```

Comment:
Are there ensembles in GCAFS?

Reply:
Not at the moment but CSL or ARL may want to test this in the near future for a future potential implementation.